src/Symfony/Component/Security/Http/AccessToken/Oidc/OidcTrait.php

```diff
@@ -24,7 +24,7 @@ trait OidcTrait
 {
     private function createUser(array $claims): OidcUser
     {
-        if (!\function_exists(\Symfony\Component\String\u::class)) {
+        if (!\function_exists(u::class)) {
```

Comment:
we need the namespace to check for the existence of the function (fixed in 862c6fbf803bd34d653699b15baadcb216708c3b)

Reply:
good catch, let me convert it into bug report:
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/7652

Reply:
I find this very curious inconsistence on PHP engine side.

https://3v4l.org/qtBRn

Reply:
deserves a bug report to php-src IMHO

Reply:
I see reasons why PHP allow for that. For `::class`, one can do whatever, like `fooisufdhsilufgvhlaiuwrgfliudfzbhsdfbsdfb::class`, and it's not evaluated if that foo..sth exists at all, therefore also not if it's a class or any other symbol like function.
It would need the compiler to already have execution context, or at least import all files... to analyse. IMHO doubtful to get in, but if you think otherwise and have better expertise on it, please consider to drive this report.